Sources/PersistenceClient/Client.swift

```diff
@@ -0,0 +1,11 @@
+import ClientModels
+import Foundation
+
+public struct PersistenceClient {
+  public var userSettings: () async -> UserSettings
+  public var setUserSettings: (UserSettings) async -> Void
```

Comment:
I think this applies to the preexisting design of Isowords too, but what is the motivation for marking `userSettings` and `setUsetSettings` as `async` when the underlying `UserDefaults` operations are all synchronous, and reasonably fast too?

Reply:
Ah, you're right about the getter not needing to be asynchronous (I missed that in the original conversation where you pointed that out :) ). The reason I made setUserSettings asynchronous was because of the discussion that took place in https://github.com/pointfreeco/swift-composable-architecture/discussions/1384 where Stephen said "but writing should be async so that it forces you to access it from within an Effect rather than calling directly in a reducer.".

With that being said, I didn't completely follow making it an AsyncStream because I haven't had experience dealing with this kind of thing. Since I've been developing with UIKit, a lot of the stuff Stephen and Brandon talk about goes over my head because SwiftUI has tons of implementations that UIKit does not (like handling binding within a reducer; something I have to mimic using Combine in a UIView/UIViewController...unless it is possible but I'm not aware) :)